pepper-apis/src/main/java/org/broadinstitute/ddp/route/GetStudyPasswordPolicyRoute.java

```diff
@@ -33,14 +33,21 @@
     public PasswordPolicy handle(Request request, Response response) {
         String studyGuid = request.params(RouteConstants.PathParam.STUDY_GUID);
         String clientId = request.queryParams(RouteConstants.QueryParam.AUTH0_CLIENT_ID);
+        String domain = request.queryParams(RouteConstants.QueryParam.AUTH0_DOMAIN);
```

Comment:
Also needs API doc change.  We could make the server more forgiving here as well by looking for a single client with the given `clientid` if `domain` is missing, and if 0 or > 1 are  found, error out.

Reply:
In light of backwards-compatibility, perhaps we can try NOT require the `domain` query param and use the one associated with the study itself, since they should both be in the same domain?

Reply:
I implemented the approach suggested by @zyme: if a `domain` is missing, we check if the `clientId` is unique. If it is, we proceed or else fail. If the `domain` is present, we use it. From the client's point of view, there is no difference until we start getting `clientId` clashes.

Reply:
It will work, but I don't think that approach is necessary for this API. That's because both the study and client should ideally belong to the same tenant/domain (if they're not, what does that mean?). So we can just use the domain from the study. We can even grab the `auth0TenantId` from the `StudyDto` object. If we can avoid changing the API (even if it's non-breaking), I think that will be better way to go so we don't have to support something that's no needed. Welcome other suggestions here.

Reply:
Well, I couldn't figure out what you mean by the domain "associated with the study itself" (the `umbrella_study` table has nothing resembling a domain). However, I used `auth0TenantId` from `StudyDto` as you suggest so there's no breaking change in this route anymore.